ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -629,7 +638,15 @@ func resourceIBMContainerVpcClusterCreate(d *schema.ResourceData, meta interface
 		}
 
 	}
-	return resourceIBMContainerVpcClusterUpdate(d, meta)
```

Comment:
As you updated return from update to read.
I'm just wondering if this will effect adding tags, kms_config and other features during creation of cluster.
Did you test this?

Reply:
Create and update operations as a whole should be isolated from one another. The create operation may perform similar tasks to the update operation, if they do, that code should be split into atomic functions that can be called from the create or update path.

Reply:
@lewiseevans,
Then can you please add tags and kms_config support in create method as well similar to taints? so that it is created along with cluster.

Reply:
added it to create and tested, pushed the test too. Created a KP instance and used it so just had to fill in vars.

--- PASS: TestAccIBMContainerVpcClusterKMSEnvvar (2102.24s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/kubernetes      2103.776s